From 9e7e45c20e6eea6dc93e3daee72d1adc9293fc7f Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 18 Dec 2019 18:36:01 +0800
Subject: [PATCH] ARM: rockchip_defconfig: disable CONFIG_KALLSYMS_ALL

before:
   text		   data	    bss	    dec		    hex	filename
12231986	4559104	2146976	18938066	120f8d2	vmlinux
after:
   text		   data	    bss	    dec		    hex	filename
12232306	3698944	2146976	18078226	113da12	vmlinux

Save about 840K.

Change-Id: I4472a77bdd2533516dd9bf5007d9507939647562
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 04c061bf7f63..6a0690b49f42 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -32,7 +32,6 @@ CONFIG_INITRD_ASYNC=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 # CONFIG_SYSFS_SYSCALL is not set
 # CONFIG_FHANDLE is not set
-CONFIG_KALLSYMS_ALL=y
 CONFIG_BPF_SYSCALL=y
 CONFIG_EMBEDDED=y
 # CONFIG_VM_EVENT_COUNTERS is not set
-- 
2.35.3

